lib/validation.js

```diff
@@ -277,6 +277,28 @@ module.exports = function (yargs, usage, y18n) {
     }
   }
 
+  var conflicting = {}
+  self.conflicts = function (key1, key2) {
+    if (typeof key1 === 'string' && typeof key2 === 'string' && key1 !== '_' && key2 !== '_') {
```

Comment:
I recommend that we don't add an special handling of `_` for the time being -- if someone complains that they tried to set `_` as a conflicting arg and it didn't work we can:

1. figure out what the heck they expected it to do, and potentially add support.
2. or, instead add a warning about this.